Dalamud/Game/Gui/ContextMenu/ContextMenuType.cs

```diff
@@ -0,0 +1,18 @@
+namespace Dalamud.Game.Gui.ContextMenu;
+
+/// <summary>
+/// The type of context menu.
+/// Each one has a different associated <see cref="MenuTarget"/>.
+/// </summary>
+public enum ContextMenuType
+{
+    /// <summary>
+    /// The default context menu.
+    /// </summary>
+    Default,
```

Comment:
historically was called GameObject by most implementations. I just copied this from Anna's previous version myself - not sure what the "right" answer is.

Reply:
@Aireil probably has thoughts

Reply:
I think default does make more sense, as it can be the crafting log or action bars for example.

Reply:
Calling it "GameObject" lead to much confusion on my part when I originally tried using the Context Menu nuget.

I definitely think "Default" is more user friendly.